src/webgl/framebuffer.js

```diff
@@ -187,15 +186,6 @@ export default class Framebuffer extends Resource {
         this._attachTexture({attachment, texture: object, layer: 0, level: 0});
       }
 
-      if (setFilteringParameters && object instanceof Texture2D) {
```

Comment:
Yeah, thats is what my understanding too, those settings should affect sampling only and should have no impact when writing to a render target.